ui/packages/catalog-ui-search/src/main/webapp/component/source-item/source-item.less

```diff
@@ -22,6 +22,19 @@
         text-overflow: ellipsis;
     }
 
+    .source-actions {
+        padding-left: 100px;
```

Comment:
I think this should be a multiple of `@minimumSpacing`

Reply:
I changed it to `padding-left: ~'calc(2 * @{minimumButtonSize})';`